### PR TITLE
Mrc-5662 Run submit

### PR DESCRIPTION
**Pull request by absternator**
This PR continues on from #115 and contains the run button and submission of run. After the run button is clicked and is successful, user will get redirected to logs page for that run (taskId).

Testing:
1. on runner page fill out fields and click run. Shold get redirected to page with task logs for that task id

### CODE REVIEW

**Review by M-Kusumgar — APPROVED** [review]
LGTM!

File: api/app/src/main/kotlin/packit/controllers/RunnerController.kt [hunk 1]
```diff
@@ -4,64 +4,53 @@ import org.springframework.http.ResponseEntity
...
-import packit.model.dto.SubmitRunInfo
+import packit.model.dto.*
 import packit.service.RunnerService
 
 @RestController
 @RequestMapping("/runner")
 @PreAuthorize("hasAuthority('packet.run')")
-class RunnerController(private val runnerService: RunnerService)
-{
+class RunnerController(private val runnerService: RunnerService) {
```
Comment (EmmaLRussell) [inline api/app/src/main/kotlin/packit/controllers/RunnerController.kt:13]: What's with the formatting changes in this file - they're not consistent with the other modules in the project I don't think.. ?

**Review by EmmaLRussell — COMMENTED** [review]
Guess I missed the boat on this one too! 😢 

It seems to make a lot of API calls when you hit run even when the form isn't valid. Run button probably shouldn't be enabled in that case?
image (13)

**Comment by absternator** [comment]
> Guess I missed the boat on this one too! 😢
> 
> It seems to make a lot of API calls when you hit run even when the form isn't valid. Run button probably shouldn't be enabled in that case?

could not replicate this

**Comment by EmmaLRussell** [comment]
> > Guess I missed the boat on this one too! 😢
> > It seems to make a lot of API calls when you hit run even when the form isn't valid. Run button probably shouldn't be enabled in that case?
> 
> could not replicate this

Weird! I'll try to show it to you next time we're both in the office.

**Pull request merged**